test/compile/mark/bar.test.ts

```diff
@@ -35,7 +35,7 @@ describe('Mark: Bar', function() {
     });
   });
 
-  describe('horizontal, with log', function() {
+  describe('vertical, with log', function() {
```

Comment:
So if I understand correctly, this test was correct.  The new change is wrong.

You might wanna check if we have similar test for tick.test.ts, if not you should write one.